### DISCUSSION TITLE
Type of error in useAsyncData hook is <true | Error>

### DISCUSSION THREAD

**Discussion by feelixe** [nuxt/nuxt#16530] [Questions] [answered]
In VSCode, when I write
```
const { error } = useAsyncData("hello", () => $fetch("/api/hello/"))
```
`error` has the type of `Ref<true | Error>`.

According to the [docs ](https://v3.nuxtjs.org/api/composables/use-async-data/#type) I'm expecting it to have the type of `Ref<ErrorT | null>`.

I'm unable to recreate it with a Stackblitz since you can view the types there.

It's possible this only occurs on my computer by some caching or similar which I have tried to clear however. If anyone could try to recreate that would be very helpful!

**Comment by danielroe** [comment, marked as answer]
This will be resolved shortly with https://github.com/nuxt/framework/pull/8521
